docs/admin/upgrading_fedora.rst

```diff
@@ -12,23 +12,23 @@ usb``, ``work``, and ``vault`` . In Qubes 4.1.2, these VMs are based on a
 Fedora 37 template, which reached end-of-life on December 5, 2023.
```

Comment:
The title of this file is still "Upgrading to Fedora 38"; I'll just quickly amend and merge.

Reply:
dangit and thank you!